production/technical_userguide.md

```diff
@@ -67,15 +65,42 @@ Client Secret:
 * `Accept: application/json`
 * `Authorization: <Encoded Basic authentication>`
 
-In the following cURL command, notice that we have concatenated the base64 encoding of the 'Client ID":" Client Secret' as the argument to the -H flag.
```

Comment:
We still need to ensure that, somewhere, this is clear.

Reply:
Accepted; resolved.

Reply:
Accepted; resolved.